README.md

```diff
@@ -66,7 +66,7 @@ $ bower install angularfire --save
 
 * [Quickstart](docs/quickstart.md)
 * [Guide](docs/guide/README.md)
-* [API Reference](https://angularfire.firebaseapp.com/api.html)
+* [API Reference](docs/reference.md)
```

Comment:
No, this page is at the root of the repo, so you don't need the leading `/`. It's just like doing `require()` in Node. Relative paths are relative to the current file and absolute paths are relative to the root of the repo.